Specs/DateTime/Japanese/TimePeriodParser.json

```diff
@@ -789,14 +765,13 @@
     "Context": {
       "ReferenceDateTime": "2017-12-01T13:37:00"
     },
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
         "Text": "午前11時から11時50分まで",
         "Type": "timerange",
         "Value": {
-          "Timex": "(T11,T11:50,PT50M)",
+          "Timex": "(T11,T11:50,PT0H50M)",
```

Comment:
This is technically correct. But the previous timex is preferrable.

Reply:
we were updating this, but since it breaks many cases in Chinese we decided to follow Chinese approach. 
Should we update Chinese test cases to follow standard format (without '0H')?

Reply:
We should fix Chinese and Korean too. The code that generates the timex should be the same (centralized object/method/...).

Reply:
modified and updated specs in Chinese and Japanese